### PR TITLE
feat: add fr_CA locale

### PR DESCRIPTION
**Pull request by liufenghua808**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    fr_CA       |
| 🇨🇳 Chinese |    法语（加拿大）       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
我拷贝你的内容创建了一个新的PR #29748

File: components/upload/__tests__/__snapshots__/uploadlist.test.js.snap [hunk 1]
```diff
@@ -28,8 +28,8 @@ exports[`Upload List handle error 1`] = `
     class="ant-upload-list ant-upload-list-text"
   >
     <div
-      class="ant-upload-list-text-container ant-upload-animate-appear ant-upload-animate-appear-start ant-upload-animate"
-      style="height: 0px; opacity: 0;"
+      class="ant-upload-list-text-container ant-upload-animate-appear ant-upload-animate-appear-active ant-upload-animate"
```
Comment (afc163) [inline components/upload/__tests__/__snapshots__/uploadlist.test.js.snap:31]: 这里把 CI 搞挂了。

**Comment by afc163** [comment]
ping @liufenghua808

**Comment by liufenghua808** [comment]
> ping @liufenghua808

> 这个文件的更新是执行 npm test -- -u之后自动更新的，不过我看CI报错也怀疑和这个更新有关

**Comment by liufenghua808** [comment]
> 我把这个PR关闭可以吗，只跟踪 > https://github.com/ant-design/ant-design/pull/29748

**Pull request closed without merging**